tests/test_jira_api.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+from src.jira_api import *
+
+api = JiraApi()
+
+
+class TestSuite:
+
+    @allure.tag('api')
+    @allure.title("Login via API")
+    @pytest.mark.api
+    @pytest.mark.parametrize("response,expected", [
+        (api.login("OlegAverkin", "fake"), [401, "AUTHENTICATED_FAILED"]),
+        (api.login("fake", "OlegAverkin"), [401, "AUTHENTICATED_FAILED"]),
+        (api.login("OlegAverkin", "OlegAverkin"), [200, "OK"]), ])
+    def test_login_to_jira(self, response, expected):
+        assert response == expected
+
+    @allure.tag('api')
+    @allure.title("Post issue")
+    @pytest.mark.api
+    def test_post_issue(self):
+        response = api.post_issue("OA API Issue001", "OlegAverkin", "Low")
+        assert response.status_code == 201
+
+    @allure.tag('api')
+    @allure.title("Update issue")
+    @pytest.mark.api
+    def test_update_issue(self):
+        response = api.post_issue("OA API Issue002 ", "OlegAverkin", "Medium")
+        issue_id = response.json().get('id')
+        response_update = api.update_issue(issue_id, "OA API Issue002 Updated", "OlegAverkin", "Low")
+        response_update.status_code = 204
```

Comment:
Are you sure you want to assign the value rather than check the value?

Reply:
Fixed.